composer.json

```diff
@@ -5,7 +5,7 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.3"
+        "php": ">=7.4"
```

Comment:
Why not `>= 7.3`?

Reply:
no time to support 3 versions of php :P

Reply:
So 7.3 version should be removed from travis config as well.